dali/base/dautils.cpp

```diff
@@ -3881,7 +3881,7 @@ class FileReadPropertiesUpdater : public CSimpleInterfaceOf<IFileReadPropertiesU
     }
 };
 
-IFileReadPropertiesUpdater * createFileReadPropertiesUpdater(IUserDescriptor * udesc)
+extern da_decl IFileReadPropertiesUpdater * createFileReadPropertiesUpdater(IUserDescriptor * udesc)
```

Comment:
trivial/style: we don't tend to put decl spec on the prototypes in the cpp files. I realize there are some here (in this file) but most don't.